### PR TITLE
Build system to create css files with inline images and svgs, and svg symbol files.

### PR DESCRIPTION
**Pull request by MannyC**
The build_example.html file has example usage of the built files.

The class names could probably be abbreviated.

I haven't tested under Windows, but I believe it should work.   The directory structure and the attempt to be cross platform make the build statements pretty long winded, so gulp may be a better option.  

I think the external xlink svg method requires viewBoxes on the symbols, which requires the original svgs to have viewBoxes defined on them. Most of the svgs do not have viewBoxes, so it's possible this method isn't usable.

### CODE REVIEW

**Comment by cjdowner** [comment]
Thanks for the contribution. Sorry for my ignorance - I'm not a web developer by day (or night), what does this PR include exactly?

**Comment by MannyC** [comment]
It's a set of scripts that will allow you to type
`npm run build`
and it will create a `dist` folder containing 

- single css files for each size and color pairing with the png images inlined in them (so you do not need the actual png files, just a single (large) css file)
- svg files for each each color that contain all of the relevant svgs as `symbol`s, which allow you to do a similar thing with svgs (one svg file, many images)

So basically it's combining all the images and svgs into single usable files.  I'd suggest you try running it to see how it goes, I put some brief instructions in the README file.

This could all be automated so that you would push to github and then another service would build the files and publish them on npm.  I could set most of that up, but I think you'd need to get to grips the basics of what this is doing first.

**Comment by saniales** [comment]
I double checked the repo, I am a dev. It is mergeable :)

**Comment by MannyC** [comment]
I have flattened out the directory structure in the `build` folder and changed the names of the files to describe their contents.

**Comment by cjdowner** [comment]
OK, I'll merge this in. Thank you for the contribution!
